### PR TITLE
DAOS-6715 test: Online reintegration (More tests)

### PR DESCRIPTION
**Pull request by rpadma2**
Test-tag-hw-medium: pr,hw,medium,ib2 online_reintegration

Summary
-  Re-enable the some of the skipped tests
-  Add mdtest application running in parallel along with IOR.
-  Test with different object class
-  Test with aggregation on.
-  Test without enabling checksum

Signed-off-by: rpadma2 <ravindran.padmanabhan@intel.com>

### CODE REVIEW

File: src/tests/ftest/osa/osa_online_reintegration.py [hunk 1]
```diff
@@ -55,7 +53,8 @@ def daos_racer_thread(self):
         self.daos_racer.run()
 
     def run_online_reintegration_test(self, num_pool, racer=False,
-                                      server_boot=False):
+                                      server_boot=False,
+                                      oclass=None):
```
Comment (shimizukko) [inline src/tests/ftest/osa/osa_online_reintegration.py:57]: You might want to update the argument descriptions.

**Review by shimizukko — DISMISSED** [review]

File: src/tests/ftest/osa/osa_online_reintegration.py (hunk 1 above)
Comment (rpadma2) [reply src/tests/ftest/osa/osa_online_reintegration.py:57]: Done.

**Review by ravalsam — APPROVED** [review]

File: src/tests/ftest/osa/osa_online_reintegration.py [hunk 2]
```diff
@@ -63,6 +63,8 @@ def run_online_reintegration_test(self, num_pool, racer=False,
                           some data in pool. Defaults to False.
             server_boot (bool) : Perform system stop/start on a rank.
                                  Defults to False.
+            oclass (str) : daos object class string (eg: "RP_2G8").
```
Comment (shimizukko) [inline src/tests/ftest/osa/osa_online_reintegration.py:66]: Also, "data (bool)" needs to be replaced with racer.

**Review by shimizukko — DISMISSED** [review]

**Comment by rpadma2** [comment]
Disabled online reintegration test (with aggregation enabled) and object class due to JIRA (DAOS-7996).

**Review by shimizukko — APPROVED** [review]

**Comment by rpadma2** [comment]
Known issues:
ior_small : https://jira.hpdd.intel.com/browse/DAOS-8233
poolcreate: https://jira.hpdd.intel.com/browse/DAOS-8229

**Review by shimizukko — APPROVED** [review]

**Review by ravalsam — APPROVED** [review]

**Comment by sylviachanoiyee** [comment]
DAOS-7996 is resolved. Maybe we need to re-test and skip tests?

**Comment by rpadma2** [comment]
> DAOS-7996 is resolved. Maybe we need to re-test and not skip tests?

Can we enable that test alone separately in a PR? I don't want to gate the other tests for enabling a single test...

**Comment by sylviachanoiyee** [comment]
> > DAOS-7996 is resolved. Maybe we need to re-test and not skip tests?
> 
> Can we enable that test alone separately in a PR? I don't want to gate the other tests for enabling a single test...

OK, please push another PR to re-enable
test_osa_online_reintegration_with_aggregation
test_osa_online_reintegration_oclass

**Pull request merged**